packages/wdio-config/src/lib/ConfigParser.ts

```diff
@@ -39,20 +122,27 @@ export default class ConfigParser {
             throw new Error('addConfigFile requires filepath')
         }
 
-        const filePath = path.resolve(process.cwd(), filename)
+        const filePath = path.isAbsolute(filename) ? path.normalize(filename) : path.resolve(this._pathService.getcwd(), filename)
```

Comment:
I actually meant to leave this out, as this is a change, but I noticed it was used elsewhere and seemed like it might be safer. Wanted to highlight it to make sure you saw :)

Reply:
👍